src/shared/varlink-io.systemd.Udev.c

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include "varlink-io.systemd.Udev.h"
+
+static SD_VARLINK_DEFINE_METHOD(
+                SetChildrenMax,
```

Comment:
for new varlink apis please add basic comments right-away. It's high on my todo list to also add comments for the pre-existing varlink apis, but so far didn't find the time to.